braga/README.md

```diff
@@ -0,0 +1,655 @@
++ **_Secure_ Building Access** membership card and/or NFC-enabled mobile phone.
+  all exterior doors will have security cameras for the safety of members.
++ **Bunk Beds** - are there already _great_ quality beds we can use
+or do we need to spec/design and order our own from scratch.
++ Investigate fresh produce (_wholesale_) market for sourcing fruit & veg
+with a bulk discount.
++ Should we make all fruit & veg "free" for members?
++ A clear code-of-conduct must be drawn up using dwyl's existing one.
+It _must_ be _clear_ to all what constitutes acceptable behaviour
+and what the grounds are for membership (_and the revocation thereof_)
+e.g: knowingly breaking any local law will lead to membership termination
+because each member is an ambassador/representative of the entire org/team.
+
+## 2. Workspace
+
+Our plan is to create _ideal_ ("***deep***") working conditions
+for the **32 people** staying with us.
+
+Considering that we currently have 26+ people
+working in the **140m**<sup>2</sup> at Palmer's road,
+we expect people to be a **lot more _comfortable_**
+with **300m**<sup>2</sup>.
+
+### Desks
+
+We will make **32 sit/stand desks** available for people to use.
+The desks will be primarily in the "**Main WorkSpace**"
+which will have **18 desks**.
+The remaining desks will be
+
+### Dimensions
+
+Each desk will measure 100 x 70cm
+(_this is **compact** but **comfortable** for a Laptop,
+  Gas-lift mounted Screen, notepad and other personal afects_)
+
+### Sit/Stand
+
+Desks can be  have a vertical range of between 65cm and 125cm
+(_this **exceeds** the range of motion of an IKEA Bekant sit/stand desk_)
+
+Desks will be on castors and can be moved into different layouts.
+We can easily switch between
+"classroom" format (_everyone facing the projector screen_)
+and "deep work" (_everyone facing the walls to avoid distraction_)
+or "team work" (_where desks are clustered to promote collaboration_).
+
+
+### Day Visitors
+
+Once we are _well established_ we will encourage other people
+to visit during the day to learn & collaborate with us.
+
+The "main room" will be reserved for focussed work
+as visitors will inevitably distract others.
+
+Similar to "Google Campus" where _anyone_ can show up and work.
+People who are _members_ of dwyl can come and work in the house/gardens
+and use free WiFi.
+
+### No Working in the Kitchen or Dining Room during meal time(s)
+
+No laptops are allowed in the Kitchen/Dining room;
+these are areas for eating and casual conversations, not staring at screens.
+
+Tablets/phones are permitted to _demo_ work
+as part of informal discussions
+and for checking recipes in the kitchen
+but people should not sit at the table messaging or watching videos.
+
+
+## 3. Creative Technology Rapid Learning Workshops (`CTRL+W`)
+
+The purpose of `CTRL+W` is to develop and share creative/technology knowledge.
+
+Between **16:00 - 19:00** we will hold _**daily**
```

Comment:
Agreed. feel free to _research_ this by attending some High-Intensity Language Training (HILT). 
You remember our Mandarin teach telling us about it, right?
8am - 8pm every day for 3months...
BTW: all workshops are Optional...